svc/qrcodes/service.go

```diff
@@ -54,11 +60,18 @@ func (s *Service) GetDataByQRCodeID(ctx context.Context, qrcodeID uuid.UUID) (in
 	if now.After(qrcodeData.ExpiresAt) {
 		return nil, ErrQRCodeExpired
 	}
+	if qrcodeData.RewardAmount.Float64 > 0 {
+		err := s.rs.AddDepositTransaction(ctx, userID, qrcodeID, "qrcodes", qrcodeData.RewardAmount.Float64)
```

Comment:
Move qrcodes to const.

Reply:
// RelationTypeQrcodes indicates that relation type "qrcodes".
	RelationTypeQrcodes = "qrcodes"